server/src/main/java/org/elasticsearch/action/admin/indices/close/ShardCloseRequest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.action.admin.indices.close;
+
+import org.elasticsearch.action.support.replication.ReplicationRequest;
+import org.elasticsearch.index.shard.ShardId;
+
+public class ShardCloseRequest extends ReplicationRequest<ShardCloseRequest> {
```

Comment:
can we just inline this in the transport action file? I don't see the need for another file.

Reply:
Ok